app/scripts/directives/pubiframe.js

```diff
@@ -49,6 +49,22 @@ angular.module('oncokbApp')
             },
             controller: function($scope) {
                 $scope.pubs = [];
+                $scope.loadedData = false;
+                $scope.isIframeDisabled = function(url) {
+                    try {
+                        $http.get(url).success(function (response) {
```

Comment:
Maybe the Options is better than the get?

Reply:
I checked the AngularJS documents and $http doesn't have Options method like $http.options, so I added method type as a parameter in the function.